packages/arb-token-bridge-ui/src/components/common/Dialog.tsx

```diff
@@ -107,7 +107,8 @@ export function Dialog(props: DialogProps) {
         open={props.isOpen}
         initialFocus={props.initialFocus || cancelButtonRef}
         onClose={() => props.onClose(false)}
-        className="fixed inset-0 z-50 flex md:items-center md:justify-center md:bg-[rgba(0,0,0,0.6)]"
+        // z-index is set to 2147483647 because rainbowkit's modal is 2147483646
+        className="fixed inset-0 z-[2147483647] flex md:items-center md:justify-center md:bg-[rgba(0,0,0,0.6)]"
```

Comment:
We should update our rainbowikit dialog to a lower z-index instead of changing our z-index to this arbitrary number.
Can get this element by CSS `aria-labelledby="rk_connect_title"` and force it's z-index in CSS.

Reply:
I would disagree, `aria-labelledby="rk_connect_title"` could break quite easily, also having everything in tailwind is easier to debug IMO than having part in tailwind, and part in CSS

Reply:
i wouldn't want to hackily override their CSS unless it's customisable